packages/financial-templates-lib/index.js

```diff
@@ -7,9 +7,11 @@ module.exports = {
   ...require("./src/helpers/GasEstimator"),
   ...require("./src/logger/Logger"),
   ...require("./src/logger/SpyTransport"),
+  ...require("./src/price-feed/UniswapPriceFeed"),
```

Comment:
additional exports that are needed to test the trader correctly.

Reply:
Weird that this wasn't exported previously...